BrowserKit/Sources/ComponentLibrary/Buttons/LinkButton.swift

```diff
@@ -8,27 +8,23 @@ import UIKit
 public class LinkButton: ResizableButton, ThemeApplicable {
     private struct UX {
         static let buttonCornerRadius: CGFloat = 13
-        static let buttonVerticalInset: CGFloat = 12
-        static let buttonHorizontalInset: CGFloat = 16
     }
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
         backgroundColor = .clear
         titleLabel?.adjustsFontForContentSizeCategory = true
-        contentEdgeInsets = UIEdgeInsets(top: UX.buttonVerticalInset,
-                                         left: UX.buttonHorizontalInset,
-                                         bottom: UX.buttonVerticalInset,
-                                         right: UX.buttonHorizontalInset)
     }
 
     public func configure(viewModel: LinkButtonViewModel) {
         accessibilityIdentifier = viewModel.a11yIdentifier
         setTitle(viewModel.title, for: .normal)
         titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
                                                                   size: viewModel.fontSize)
+        contentEdgeInsets = viewModel.contentEdgeInsets
```

Comment:
We have some warnings related to `contentEdgeInsets`, could we look into how to fix this at the same time 🤔 ?

Reply:
Hey @lmarceau,  I have used UIButton.Configuration to fix this, please have a look.